ui/dashboardApp/index.ts

```diff
@@ -23,8 +24,33 @@ import AppInstanceProfiling from '@lib/apps/InstanceProfiling/index.meta'
 import AppClusterInfo from '@lib/apps/ClusterInfo/index.meta'
 import AppSlowQuery from '@lib/apps/SlowQuery/index.meta'
 
-async function main() {
-  client.init()
+type AppOptions = {
+  isPortal?: boolean
+  token?: string
+  lang?: string
+  hideNav?: boolean
+}
+
+const defOptions: AppOptions = {
+  isPortal: false,
+  token: '',
+  lang: '',
+  hideNav: false,
```

Comment:
We can treat `lang`, `hideNav` as a config, then store them in localStorage. When App is initialized, we read them from local storage. In this way, "Command+Click" will keep working.